cpp/src/parquet/file_writer.cc

```diff
@@ -292,36 +290,34 @@ class RowGroupSerializer : public RowGroupWriter::Contents {
       auto col_meta = metadata_->NextColumnChunk();
```

Comment:
Most of this initialization code seems the same as in `NextColumn`, can we refactor this?

Reply:
Hmm sure, currently they're used in different way (For buffered / un-buffered write ), I'll take a look tonight